distribution/ddf-common/src/main/resources/etc/custom.properties

```diff
@@ -46,59 +46,218 @@ org.apache.karaf.security.providers=sun.security.provider.Sun,org.bouncycastle.j
 # Allow configuration admin to update config files
 felix.fileinstall.enableConfigSave = true
 
+#
```

Comment:
Few questions on this file.

* Do we need to manually maintain those `uses` definitions or is this something that's generated somehow? If this is manually maintained, would there be a way to make sure they're always valid? Does it matter?
* Some of the definitions are indented. Is that on purpose?
* Do we expect our admins to make changes to this file? If so, does it need to be part of the cloning/migration process?